zerver/openapi/zulip.yaml

```diff
@@ -2560,21 +2560,11 @@ paths:
                           type: string
                           description: |
                             Whether the emoji has been deactivated or not.
-                        author:
+                        author_id:
+                          type: int
```

Comment:
This will give a syntax error, it should be `type: integer`.

Reply:
Done!